CMakeLists.txt

```
@@ -3,44 +3,45 @@ cmake_minimum_required(VERSION 3.0)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/cmake")
 find_package(SDL2 REQUIRED)
-include_directories(${SDL2_INCLUDE_DIR})
+if(WIN32)
```

mingw has `dirent.h`(https://github.com/Alexpux/mingw-w64/blob/master/mingw-w64-headers/crt/dirent.h)
```suggestion
if(MSVC)
```